### PR TITLE
Feat/cmp new version

### PR DESCRIPTION
**Pull request by midudev**
Iteration over CMP. Prepared to implement on the future a modal version.

Desktop:
image

Mobile:
image

### CODE REVIEW

File: components/cmp/modal/src/index.js [hunk 1]
```diff
@@ -7,22 +7,21 @@ import CmpServices from '@schibstedspain/react-cmp-services'
...
+    let domElementToOpenOnClick
     if (elementToOpenOnClick) {
       domElementToOpenOnClick = document.querySelector(elementToOpenOnClick)
-      createHandleClickElementEvent()
+      createHandleClickElementEvent(domElementToOpenOnClick)
     }
 
-    return () => createHandleClickElementEvent(false)
+    return () => createHandleClickElementEvent(domElementToOpenOnClick, false)
   })
```
Comment (juanmendez90) [inline components/cmp/modal/src/index.js]: Do we want to run the effect every time? If no I will add the second parameter of the effect, in this case, it seems that `elementToOpenOnClick` would work fine

File: components/cmp/ui/src/index.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  &-content {
+    @include media-breakpoint-up(m) {
+      max-width: 1220px;
+      padding-right: 250px;
+      position: relative;
+    }
+  }
+
+  &-title {
+    font-size: 16px;
```
Comment (danivm) [inline components/cmp/ui/src/index.scss]: `$fz-m`

File: components/cmp/ui/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+$bgc-cmp-ui: #fff !default;
+
+.sui-CmpUi {
+  /* Cmp is an exception over the MoleculeNotification with some specific styles
+     as we want to be the same across verticales, we're styling directly SUI.
+     But don't do that on your home. */
+  .sui-MoleculeNotification {
+    background: $bgc-cmp-ui;
+    box-shadow: 0 0 8px 2px rgba(0, 0, 0, 0.05);
+    padding-bottom: 16px;
```
Comment (juanmendez90) [inline components/cmp/ui/src/index.scss]: $p-l

File: components/cmp/ui/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    @include media-breakpoint-up(m) {
+      max-width: 1220px;
+      padding-right: 250px;
+      position: relative;
+    }
+  }
+
+  &-title {
+    font-size: 16px;
+    margin: 0 0 8px;
```
Comment (juanmendez90) [inline components/cmp/ui/src/index.scss]: $m-m

File: components/cmp/ui/src/index.scss [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+    @include media-breakpoint-up(m) {
+      font-size: 24px;
+      margin: 0 0 16px;
```
Comment (juanmendez90) [inline components/cmp/ui/src/index.scss]: $p-l

File: components/cmp/ui/src/index.scss [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    line-height: 18px;
+    display: block;
+    padding-bottom: 16px;
+
+    @include media-breakpoint-up(m) {
+      color: $c-black;
+      font-size: 16px;
+      line-height: 1.5;
+      padding-bottom: 24px;
+      padding-top: 16px;
```
Comment (juanmendez90) [inline components/cmp/ui/src/index.scss]: $p-l

File: components/cmp/ui/src/CmpUi/index.js [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+  const [showNotification, setShowNotification] = useState(false)
+  const [showPartners, setShowPartners] = useState(false)
+  const containerDOMEl = useRef()
+
+  const _handleAccept = async () => {
+    const {
+      purposeConsents,
+      vendorConsents
+    } = await getPurposesAndVendors.execute()
+    await sendConsents.execute({purposeConsents, vendorConsents})
```
Comment (davecarter) [inline components/cmp/ui/src/CmpUi/index.js]: Maaagiiiic 😍 😂

File: components/cmp/ui/src/CmpUi/index.js [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+  }
+
+  /**
+   * This handler is used everytime the user clicks on something in the web
+   * in order to assume he's accepting our CMP if he's not trying to configure it and
+   * he's just navigating the website
+   */
+  const _handleClickOnDocument = ({target}) => {
+    const isClickableElement =
+      target.closest('a') || target.closest('button') || target.closest('input')
```
Comment (igncalleja) [inline components/cmp/ui/src/CmpUi/index.js]: Internet Explorer 11 has no support for Element.closest(). A polyfill is available though: 

https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill

Comment (igncalleja) [reply components/cmp/ui/src/CmpUi/index.js]: PR for it: https://github.com/SUI-Components/sui/pull/644

**Pull request merged**